backend/Posts/views.py

```diff
@@ -9,7 +9,7 @@
 
 # Able to create and view the post, should only handles POST requests to create a post
 class CreatePost(generics.CreateAPIView):
-    queryset = Post.objects.all()
+    #queryset = Post.objects.all()
```

Comment:
again this looks like an issue can we leave any changes to these files out of your pr?